sources/backend.c

```diff
@@ -106,6 +106,15 @@ static inline int od_backend_startup(od_server_t *server,
 {
 	od_instance_t *instance = server->global->instance;
 	od_route_t *route = server->route;
+
+	if (client->rule->ldap_storage_user_attr) {
```

Comment:
Why do we need this changes? client route id should not change during client lifecycle, because user-server routing uses route username for server connection pooling and other logic

Reply:
look for sources/router.c:334, i think we can remove this logic from here and add to router.c